src/components/study-pane.js

```diff
@@ -709,6 +731,7 @@ const StudyPane = (props) => {
         loadNetwork,
         loadGeoData,
         loadTree,
+        checkStudyExists,
```

Comment:
add in upper note that `checkStudyExists` don't change as well. It's alos true for `loadTree`